### PR TITLE
Revert "Homepage"

### PR DESCRIPTION
**Pull request by pupupupupupupupupu**
Reverts kalviumcommunity/ComicCity#3

### CODE REVIEW

File: front-end/src/index.js [hunk 1]
```diff
@@ -3,15 +3,13 @@ import ReactDOM from 'react-dom/client';
...
-// import Footer from './components/footer'
-// import Navbar from './components/navbar'
-// import HomePage from './components/home';
+import Footer from './components/footer'
+import Navbar from './components/navbar'
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
```
Comment (PatelNeelMahesh) [inline front-end/src/index.js:12]: implement react-router here.

File: front-end/src/components/navbar.js [hunk 2]
```diff
@@ -109,33 +110,28 @@ const Navbar = () => {
 
         <div className="main">
           <div className="user">
-            <i className="bx bxs-user"></i>Sign In
+            <i class="bx bxs-user"></i>Sign In
```
Comment (PatelNeelMahesh) [inline front-end/src/components/navbar.js:113]: In react you can't use class keyword for css, you have to use "className".

File: front-end/src/App.js [hunk 3]
```diff
@@ -1,9 +1,227 @@
...
+          </div>
+        </div>
+
+        <div><Carousel /></div>
+        <div><PopularComics /></div>      
+        <div><LastRead /></div>
+
+      </div>
+
+      <footer><Footer /></footer>
```
Comment (PatelNeelMahesh) [inline front-end/src/App.js:50]: No need to add footer tag here, without it you can add <Footer/> component.

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
Change the PR description and do the required changed based on the feedback.

File: front-end/src/index.js (hunk 1 above)
Comment (pupupupupupupupupu) [reply front-end/src/index.js:12]: Yes I'll

File: front-end/src/components/navbar.js (hunk 2 above)
Comment (pupupupupupupupupu) [reply front-end/src/components/navbar.js:113]: Ok

File: front-end/src/App.js (hunk 3 above)
Comment (pupupupupupupupupu) [reply front-end/src/App.js:50]: Okay :)

**Pull request merged**